src/modules/75-cf/strings/strings.en.yaml

```
@@ -391,10 +391,11 @@ auditLogs:
       bucketBy: Default serve updated for bucket by '{{bucketBy}}'
       variation: Default serve updated with variation '{{variation}}'
     addTargetsToVariationTargetMap: Target '{{target}}' was added for Variation '{{variation}}'
+    addTargetToFlagsVariationTargetMap: Target '{{target}}' was added for Variation '{{variation}}'
```

is there a better way to achieve this than having two with the same value? I tried using `addTargetsToVariationTargetMap ` for both cases but this felt incorrect.

---

I *think* this could do the same job without duplication.
```suggestion
    addTargetToFlagsVariationTargetMap: {{ $.cf.auditLogs.events.addTargetsToVariationTargetMap }}
```

Not very pretty though. You could just use the same string id for both, but change it to be a little more generic... maybe `targetAddedForVariation`

---

Have used the latter suggestion

---

Yeah, I think that's the better idea. Referencing is good in some situations, but it feels kinda forced here.